src/components/Search/search.css

```diff
@@ -0,0 +1,38 @@
+.Search {
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 50px;
+    margin-left: 150px;
+    margin-right: 150px;
```

Comment:
`margin: 0 150px 50px`

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/margin#:~:text=When%20three%20values%20are%20specified,in%20that%20order%20(clockwise).

Reply:
fixed